backend/app/Http/Controllers/AccountController.php

```diff
@@ -30,4 +31,19 @@ public function show(Account $account)
     {
         return AccountResource::make($account);
     }
+
+    public function store(User $user, AccountPostRequest $request)
+    {
+        $payload = $request->validated();
+
+        $account_type = array_search($payload['account_type'], config('enums.account_type')) + 1;
+
+        $account = Account::create([
+            'user_id' => $user->id,
```

Comment:
make use of Auth::id() since this is for authenticated user. or is there a reason why we need to put the id in the api?